test.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require 'vendor/autoload.php';
+
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+// Initialize template engine https://twig.symfony.com/
+$loader = new FilesystemLoader('templates');
+$twig = new Environment($loader, [
+    'cache' => 'cache/twig',
```

Comment:
Remove this line twig won't use cache it will be easier to work with

Reply:
@HenintsoaHARINORO  by removing this line I meant lint number 10.
There is key in array named 'cache' you just need to remove `'cache' => 'cache/twig',
 
`

Reply:
Ooh,OKay